### ISSUE TITLE
App Configuration `client.list_configuration_settings()` unexpected keyword argument 'keys'

### ISSUE THREAD

**Issue by yangsiyu007** [Azure/azure-sdk-for-python#12775]
- **Package Name**: 
azure.appconfiguration

- **Package Version**: 
1.0.0

- **Operating System**:
Mac

- **Python Version**: 
3.6.10

**Describe the bug**
Following the example for listing configuration settings (https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/appconfiguration/azure-appconfiguration#list-configuration-settings), the for-each loop gives the error 
```
TypeError: request() got an unexpected keyword argument 'labels'
```
suggesting that it cannot be iterated as shown.

**To Reproduce**

```
# set up client
client = AzureAppConfigurationClient.from_connection_string(connection_str)

# example given in the documentation
filtered_listed = client.list_configuration_settings(
    labels=["*Labe*"], keys=["*Ke*"]
)
for item in filtered_listed:
    pass
```

**Expected behavior**
Be able to run through the for-each loop without erroring. Please show a working example of how this iterator should be used.

**Comment by xiangyan99** [comment]
Thanks for reporting the issue.

We have a fix for it and it will be available in next release.

**Comment by xiangyan99** [comment]
Fixed in #12784
